std_msgs/Header header

float64 joint_pos_0
float64 joint_pos_1
float64 joint_pos_2
float64 joint_pos_3
float64 joint_pos_4
float64 joint_pos_5

geometry_msgs/Point end_pos
geometry_msgs/Quaternion end_quat

geometry_msgs/Point cam_pos
geometry_msgs/Quaternion cam_quat